intro-HPC/ch_troubleshooting.tex

```diff
@@ -0,0 +1,58 @@
+\chapter{Troubleshooting}
+\label{ch:troubleshooting}
+
+
+
+\section{Walltime issues}
+If you get from your job output error a message similar to this:
+
+\begin{prompt}
+ =>> PBS: job killed: walltime 146 exceeded limit 120
+\end{prompt}
+
+You job has exceeded the requested walltime. Probably the job requires more time to be executed.
+See section~\ref{sec:specifying-walltime-requirements} for more information about how to request the walltime in an accurate way.
+If you need more than \strong{72h} to execute your job you \strong{must} use \emph{checkpointing} in that case.
```

Comment:
`72h` -> `72 hours` (and this value is site-specific, so you may need to reword that to mention `maximum amount of walltime` + a reference?)